valohai_yaml/schema/endpoint.yaml

```diff
@@ -44,35 +44,7 @@ properties:
     description: Kubernetes node selector specifies which nodes the deployment should be scheduled to.
     pattern: "="
   resources:
-    type: object
-    description: Resource requirements and limits for the endpoint.
-    additionalProperties: false
-    properties:
-      cpu:
-        type: object
-        description: CPU resource requirements and limits in vCPU counts e.g. 0.1 is 10% of one CPU.
-        additionalProperties: false
-        properties:
-          min:
-            type: number
-          max:
-            type: number
-      memory:
-        type: object
-        description: Memory requirements and limits in MB e.g. 100 is 100 MB.
-        additionalProperties: false
-        properties:
-          min:
-            type: number
-          max:
-            type: number
-      devices:
-        type: object
-        description: "Devices required e.g. 'nvidia.com/gpu: 1' for one NVIDIA GPU."
-        patternProperties:
-          "^.+$":
-            type: number
-            minimum: 1
+    "$ref": "./execution-resources.json"
```

Comment:
would name it something else as it's not always Execution we are talking about, in Deployment Endpoint context, it's not an Execution we are running, but a Deployment Version Endpoint (effectively a Kubernetes Deployment with a number of pods and containers)

maybe just `resources.json` as there is nothing else that name would clash with? Or `workload-resources.json` to lessen potential confusion why something Execution is used in Deployment Endpoints?

Reply:
Workload resources sounds good to me – I started with just `resources` but thought that sounded a bit too generic. Agree that if this is not just for executions, the current name is misleading – will change!